specification/maps/data-plane/Weather/preview/1.0/weather.json

```diff
@@ -1905,8 +2383,9 @@
           "description": "Level that the index value falls under, represented by an integer. This value can be 1 through 5 and should be used in combination with the `ascending` flag because it can differ among indices. For example, the following values apply for Mosquito Activity: Low=1, Moderate=2, High=3, Very High=4, and Extreme=5."
         },
         "ascending": {
+          "x-ms-client-name": "isAscending",
           "type": "boolean",
-          "description": "Describes the direction of the `value` and `categoryValue`. For example, when `ascending`=True, the poorest index value is 0 and the best index value is 10. When `ascending`=True, the poorest index value is 10 and the best index value is 0."
+          "description": "Describes the direction of the `value` and `categoryValue`. For example, when `is_ascending`=True, the poorest index value is 0 and the best index value is 10. When `is_ascending`=True, the poorest index value is 10 and the best index value is 0."
```

Comment:
[Rest docs](https://docs.microsoft.com/en-us/rest/api/maps/weather/get-daily-indices#dailyindex) would become inconsistent here and in other descriptions modified...

Reply:
Should we just change the rest docs separately.

Reply:
Rest docs also use the `description` here so you've already changed them.
I think the current status is as follow:

- API response: `ascending` (in production, cannot be changed)
- Rest docs: `is_ascending` (inconsistent with API response)
- Generated client and docstrings: `is_ascending` 

Either we don't rename `ascending`, or we overwrite the docstrings in the convenience layer.
@ambientlight @daniel-rocha Any thoughts on this?

Reply:
You could also simply reword this so it doesn't reference the name. 
"When set to true, the poorest..."